tests/testTriggers.go

```diff
@@ -628,6 +628,11 @@ func endLongevityTest() {
 func updateLongevityStats(name, eventStatName string, dashStats map[string]string) {
 	name = strings.Split(name, "<br>")[0] //discarding the extra strings attached to name if any
 	version, err := Inst().V.GetDriverVersion()
+	product := "px-enterprise"
+	if eventStatName == stats.AsyncDREventName || name == stats.MetroDREventName || name == stats.StorkApplicationBackupEventName {
```

Comment:
It should be either eventStatName or name used consistently across the conditions, right?

Reply:
Have grouped triggers in eventName so that we dont need to put all here. This should be ok.